### PR TITLE
XCPMD boot dep update [OXT-434] and selinux policy for default rules [OXT-449]

### PR DESCRIPTION
**Pull request by jctemkin**
Required by [xctools PR29](https://github.com/OpenXT/xctools/pull/29).

### CODE REVIEW

File: recipes-security/refpolicy/refpolicy-mcs-2.20141203/policy/modules/services/xenpmd.fc [hunk 1]
```diff
@@ -18,5 +18,7 @@
 #
 #############################################################################
 
-/usr/sbin/xcpmd		--	gen_context(system_u:object_r:xenpmd_exec_t,s0)
-/var/run/xcpmd\.pid    --      gen_context(system_u:object_r:xenpmd_var_run_t,s0)
+/usr/sbin/xcpmd                -- gen_context(system_u:object_r:xenpmd_exec_t,s0)
+/var/run/xcpmd\.pid            -- gen_context(system_u:object_r:xenpmd_var_run_t,s0)
+/usr/share/xcpmd/              -- gen_context(system_u:object_r:xenpmd_etc_t,s0)
```
Comment (jean-edouard) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/policy/modules/services/xenpmd.fc]: This line is wrong, it should be:
`/usr/share/xcpmd              -d gen_context(system_u:object_r:xenpmd_etc_t,s0)`

Comment (jctemkin) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/policy/modules/services/xenpmd.fc]: Thanks, my selinux skills are not perfect yet :)

**Comment by jean-edouard** [comment]
LGTM, except for the trailing slash in the file context for `/usr/share/xcpmd`.
Depending on how the SELinux context parser works, it may or may not be an issue, but in doubt we should be consistent with the rest of the policy and remove the trailing slash.

**Comment by jctemkin** [comment]
Trailing slash removed.

**Comment by jean-edouard** [comment]
Thank you! Will merge soon.

File: recipes-security/refpolicy/refpolicy-mcs-2.20141203/policy/modules/services/xenpmd.fc [hunk 2]
```diff
@@ -18,5 +18,7 @@
 #
 #############################################################################
 
-/usr/sbin/xcpmd		--	gen_context(system_u:object_r:xenpmd_exec_t,s0)
-/var/run/xcpmd\.pid    --      gen_context(system_u:object_r:xenpmd_var_run_t,s0)
+/usr/sbin/xcpmd                -- gen_context(system_u:object_r:xenpmd_exec_t,s0)
+/var/run/xcpmd\.pid            -- gen_context(system_u:object_r:xenpmd_var_run_t,s0)
+/usr/share/xcpmd               -d gen_context(system_u:object_r:xenpmd_etc_t,s0)
+/usr/share/xcpmd/default.rules -- gen_context(system_u:object_r:xenpmd_etc_t,s0)
```
Comment (rossphilipson) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/policy/modules/services/xenpmd.fc]: Not to be a pain and I really want to get this one in but does default.rules need the period escaped? It seems to be everywhere else in those policies.

**Comment by jctemkin** [comment]
Right you are. Dot escaped.

**Pull request merged**